airflow/config_templates/default_airflow.cfg

```diff
@@ -550,6 +550,15 @@ worker_concurrency = 8
 # Example: worker_autoscale = 16,12
 # worker_autoscale =
 
+# Due to an issue with Celery, worker_prefetch_multiplier must also be set to an int
+# greater than 1 for autoscaling to work.  Be aware that this causes the number of
+# records that a worker prefetchs to be greater than the available worker processes.
+# If you have multiple workers this can cause some jobs to be unnecessarily blocked if
+# one worker prefetches tasks and those tasks are blocked by long running work then another
+# worker with available process workers can't grab those tasks.
+# https://docs.celeryproject.org/en/stable/userguide/optimizing.html#prefetch-limits
+# worker_prefetch_multiplier =
```

Comment:
```suggestion
# worker_prefetch_multiplier = 1
```

Reply:
See comment above.  I added an `example` in `config.yml` which generates
`Example: worker_prefetch_multiplier = 1`.

Reply:
I'm not sure that you can add a `default` in `config.yml` and have it be commented out when you generate `default_airflow.cfg`.

Reply:
Yes sorry I meant, it doesn't have to be commented out so as the default is 1 :)

Reply:
Ah, got it, yeah.  I had it commented out bc autoscale is commented out but the setting has independent value and like you said default is 1 anyways.

Reply:
Missed your comment until now and given that this was just merged I'll just leave it be given that functionality is there.